tesseract/tesseract_urdf/include/tesseract_urdf/geometry.h

```diff
@@ -106,7 +107,8 @@ class GeometryStatusCategory : public tesseract_common::StatusCategory
 inline tesseract_common::StatusCode::Ptr parse(std::vector<tesseract_geometry::Geometry::Ptr>& geometries,
                                                const tinyxml2::XMLElement* xml_element,
                                                tesseract_scene_graph::ResourceLocatorFn locator,
-                                               bool visual)
+                                               bool visual,
+                                               const int version)
```

Comment:
convex_mesh and sdf_mesh have the visual argument as the last one since there is a default value.

Reply:
I switched it to be consistent throughout.